src/OmniSharp.Host/Startup.cs

```diff
@@ -36,18 +38,21 @@ public Startup(IOmniSharpEnvironment env)
 
             var configBuilder = new ConfigurationBuilder()
                 .SetBasePath(AppContext.BaseDirectory)
-                .AddJsonFile("config.json", optional: true)
+                .AddJsonFile(OmniSharpConstants.OmnisharpConfigFile, optional: true)
                 .AddEnvironmentVariables();
 
             if (env.OtherArgs?.Length > 0)
             {
                 configBuilder.AddCommandLine(env.OtherArgs);
             }
 
+            // Use the global omnisharp config if there's any in the shared path
+            configBuilder.CreateAndAddGlobalOptionsFile(env);
```

Comment:
It strikes me a little weird that we'd start creating files on the user's machine without them knowing, even if it's an empty file for convenience.